source/assets/js/playground/console-utils.ts

```diff
@@ -45,36 +59,92 @@ function lineNumberFormatter(number?: number): string {
   return `${number}`;
 }
 
-export function displayForConsoleLog(item: ConsoleLog): string {
-  const data: {type: string; lineNumber?: number; message: string} = {
+function selectionLink(
+  playgroundState: PlaygroundState,
+  range?: PlaygroundState['selection']
+): string {
+  if (!range) return '';
+  return serializeState({...playgroundState, selection: range});
+}
+
+export function displayForConsoleLog(
+  item: ConsoleLog,
+  playgroundState: PlaygroundState
+): string {
+  const data: {
```

Comment:
Why store this as an object rather than distinct variables?

Reply:
I found it slightly easier to track initially, but it doesn't have a useful purpose now, removed in https://github.com/sass/sass-site/pull/1179/commits/2d7ba875d1154d13cd07cdf950058ec55826d158